core/domain/story_domain.py

```diff
@@ -15,7 +15,9 @@
 """Domain objects relating to stories."""
 
 from constants import constants
+from core.domain import skill_services
```

Comment:
story_domain should not import skill_services -- see comments on validation in the topic PR.

Reply:
Done.